tests/common/snappi/snappi_fixtures.py

```diff
@@ -517,3 +521,355 @@ def cvg_api(snappi_api_serv_ip,
     yield api
     if getattr(api, 'assistant', None) is not None:
         api.assistant.Session.remove()
+
+
+def snappi_dut_base_config(duthost_list,
+                           snappi_ports,
+                           snappi_api):
+    """
+    Geenrate snappi API config and port config information for the testbed
```

Comment:
Comments are STILL inconsistent with definition of the function. @selldinesh

Reply:
this has been resolved